.github/ISSUE_TEMPLATE/anything_else.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Anything else
```

Comment:
I wonder if we should say "Other" than "Anything else". More of a nit and not something I have strong feelings on and something we can always change later.

Reply:
done, no strong preference either (originally debated between the two lol)